Makefile

```diff
@@ -9,6 +9,14 @@ help: Makefile
 
 ## test:        run automated test suite
 test:
+	pytest --cov=yeat -m 'not long'
+
+## testlong:    run only long-running automated tests
+testlong:
+	pytest --cov=yeat -m long
```

Comment:
Quick question about `coverage`. When you break up tests, would be best to for go `coverage` since you will not be running most of the codebase? Would it make more sense to only test for coverage in `make testall`.

Currently `make testall` is at 99% coverage; however, others have lower amounts. I can see where showing coverage can still be helpful, but I feel like it's not helpful at the sometime. 🤷‍♂️

Reply:
Yeah, measuring coverage on a subset of tests doesn't give you very much insight.